src/components/network/equipment-table.js

```diff
@@ -161,38 +220,58 @@ export const EquipmentTable = ({
     );
 
     const EditableCellRender = useCallback(
-        (cellData, numeric, changeCmd, fractionDigit) => {
-            return !isLineOnEditMode(cellData.rowIndex) ||
-                cellData.rowData[cellData.dataKey] === undefined ? (
-                defaultCellRender(cellData, numeric, fractionDigit)
-            ) : (
-                <TextField
-                    id={cellData.dataKey}
-                    type="Number"
-                    className={classes.cell}
-                    size={'medium'}
-                    margin={'normal'}
-                    inputProps={{ style: { textAlign: 'center' } }}
-                    onChange={(obj) =>
-                        registerChangeRequest(
+        (cellData, numeric, changeCmd, fractionDigit, Editor) => {
+            if (
+                !isLineOnEditMode(cellData.rowIndex) ||
+                cellData.rowData[cellData.dataKey] === undefined
+            ) {
+                return defaultCellRender(cellData, numeric, fractionDigit);
+            } else if (
+                !editableEquipmentFetched ||
+                lineEdit.id !== editableEquipment?.id
+            ) {
+                return <CircularProgress size={ROW_HEIGHT} />;
+            } else {
```

Comment:
When CircularProgress is displayed, it is displayed on the left of the cell, and the height of the circle is equal to the height of the cell, which does not look very nice. Maybe it could be displayed with a smaller size (tested with half the row height and it looks better) and centered in the cell.

Reply:
no more circular progress, limits are now in data